test/cli/commands_test.py

```diff
@@ -149,20 +149,49 @@ def test__cli__command_lint_stdin(command):
     "command",
     [
         # Test basic linting
-        (lint, ["-n", "test/fixtures/cli/passing_b.sql"]),
+        (lint, ["-n", "test/fixtures/cli/passing_b.sql", "--exclude-rules", "L051"]),
```

Comment:
FYI I have excluded L051 from these tests as `passing_b.sql` contains a lone JOIN. Alternatively I can remove the lone JOIN from that test query? 😄